reference/sisdata.v1.yaml

```diff
@@ -278,27 +278,27 @@ components:
         name:
           type: string
         school:
-          $ref: '#/components/schemas/school'
+          $ref: '#/components/schemas/School'
         students:
           type: array
           items:
-            $ref: '#/components/schemas/studentReference'
+            $ref: '#/components/schemas/StudentReference'
         teachers:
           type: array
           items:
-            $ref: '#/components/schemas/teacherReference'
+            $ref: '#/components/schemas/TeacherReference'
         subjectChoices:
           type: array
           items:
-            $ref: '#/components/schemas/subject-choice'
+            $ref: '#/components/schemas/SubjectChoice'
```

Comment:
Ik zou zelf gaan voor SchoolSubject, dan ontstaat er zeker geen spraakverwarring met de StudentSubjectChoice

Reply:
I think I will do another proposal: split the student specific fields (start/end), so SubjectChoice is always the same

Reply:
Or just exchange only a ref to the subject with a start- and enddate. So you can get all the SchoolSubjects first, and with the Student-object you just reference a SchoolSubject